inference-engine/tests/functional/plugin/myriad/shared_tests_instances/import_export_tests/import_nonzero.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "import_export_tests/import_nonzero.hpp"
+#include "vpu/private_plugin_config.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+TEST_P(ImportNonZero, CompareWithRefImpl) {
+    Run();
+};
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32,
```

Comment:
There should be only these precisions to test?

Reply:
Actually this test doesn't depend on precision at all, the network should only have dynamic output in order to reproduce the initial problem. So I've finally decided to leave only one precision to remove unnecessary test cases.